openapi/data_repository_service.swagger.yaml

```diff
@@ -240,11 +226,10 @@ definitions:
         items:
           type: string
         description: |-
-          A list of strings that can be used to identify this Data Bundle.
-      system_metadata:
-        $ref: '#/definitions/SystemMetadata'
-      user_metadata:
-        $ref: '#/definitions/UserMetadata'
+          A list of strings that can be used to find this Data Bundle.
```

Comment:
I'm concerned about saying "can be used to find", since this API has no search features, and we don't yet have integration with the search API.